components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ParentSiddhiApp.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.carbon.status.dashboard.core.bean;
+
+/**
+ * Class that contains the parent siddhi application details.
+ */
+public class ParentSiddhiApp {
+    private String parentAppName;
+    private String groupName;
+    private String appName;
+    private String id;
+
+    private ParentSiddhiApp(){
```

Comment:
Unnecessary empty constructor.
No need to put this as javac will add the default for you.